streams/src/main/java/org/apache/kafka/streams/processor/internals/SourceNode.java

```diff
@@ -85,7 +85,7 @@ public void init(final InternalProcessorContext context) {
     @Override
     public void process(final K key, final V value) {
         context.forward(key, value);
-        sourceNodeForwardSensor().record();
+        forwardSensor.record();
```

Comment:
I was wondering why the forward sensor was defined as a full-blown throughput and latency sensor but then it is only recorded like a throughput sensor. I kept it as it was for `0.100-2.4` and in `latest` the sensor is removed anyways. I was just surprised.

Reply:
The docs describe only throughout metrics for `forward`.  So, I changed the definition of the forward sensor to a pure throughput sensor.